CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_twilio_source v0.3.0
+
+## 🚨 Breaking Changes 🚨
```

Comment:
We should probably call out why it's a breaking change.

Reply:
thanks Avinash! yes it was mentioned to be a breaking change in the case where users were expecting string values, but i made that more clear!